src/tracks/gosling-track/gosling-track.ts

```diff
@@ -1053,6 +1112,13 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
          *
          * */
 
+        /**
+         * This is for the HiGlass mouseMoveZoom event. However, GoslingTrack has its own way of handling mouse events.
+         */
+        mouseMoveZoomHandler() {
+            return;
+        }
+
```

Comment:
The original function is here https://github.com/higlass/higlass/blob/335f4eea6445d07baa5eda2e66077dca461b7777/app/scripts/HorizontalTiled1DPixiTrack.js#L173 

I don't think it is used since Gosling uses a different mouse events api